fonts.css

```diff
@@ -0,0 +1,7 @@
+@font-face{
+	font-family: 'Montserrat';
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/tesla_car/pullRequest?prid=5929182)